.circleci/config.yml

```diff
@@ -16,7 +16,8 @@ jobs:
 
       - run:
           command:
-            sbt compile exit
+            sbt compile
```

Comment:
1. It won't work – if you need a multiline command, you have to write it with `command: |`

2. It's better to do them as separate steps (several `run`s) so that it would be clear where the CI failed

Reply:
So, it's should be like this?
`      - run:
          command:
            sbt compile

      - run:
          command:
            sbt assembly`